js/packages/teams-ai/src/authentication/Authentication.ts

```diff
@@ -14,38 +14,43 @@ import { Application, Selector } from '../Application';
 import { MessagingExtensionAuthentication } from './MessagingExtensionAuthentication';
```

Comment:
I know you're going to hate this but unfortunately, `MessagingExtension(s)` is the incorrect terminology. `MessageExtension` is correct. Please do a rename (I asked the same for Chaoyi's PR). 

- if it's an import from botbuilder, it's fine to leave it as-is. We can't do anything about pre-existing incorrect naming.
- From here on out, any instance of message extension related names should be kept as MessageExtension, and Messaging... should be avoided as much as possible. 
- Same is applicable to samples

Sorry for not catching this earlier.

Reply:
done